field_rrule/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================
+Repetition rules
+================
+
+This module was written to offer a field type that holds rrules according to RFC 2445.
```

Comment:
Quick review looks like this complies with 5545, just need to update the ReadMe.

More info https://tools.ietf.org/html/rfc5545#appendix-A